fendermint/app/config/default.toml

```diff
@@ -11,6 +11,8 @@ snapshots_dir = "snapshots"
 contracts_dir = "contracts"
 # Builtin actor bundle CAR file.
 builtin_actors_bundle = "bundle.car"
+# In-repo actor bundle CAR file.
+actors_bundle = "actors_bundle.car"
```

Comment:
Maybe something like `custom_actors_bundle` would help highlight the difference between the two.